extension/js/common/ui/key-import-ui.ts

```diff
@@ -105,18 +105,18 @@ export class KeyImportUi {
     const attach = new AttachmentUI(() => Promise.resolve({ count: 100, size: 1024 * 1024, size_mb: 1 }));
     attach.initAttDialog('fineuploader', 'fineuploader_button', {
       attAdded: async file => {
-        let prv: OpenPGP.key.Key | undefined;
+        let prv: Key | undefined;
         const utf = file.getData().toUtfStr();
         if (utf.includes(PgpArmor.headers('privateKey').begin)) {
           const firstPrv = MsgBlockParser.detectBlocks(utf).blocks.filter(b => b.type === 'privateKey')[0];
           if (firstPrv) { // filter out all content except for the first encountered private key (GPGKeychain compatibility)
-            prv = (await opgp.key.readArmored(firstPrv.content.toString())).keys[0];
+            prv = (await KeyUtil.parse(firstPrv.content.toString()));
           }
         } else {
-          prv = (await opgp.key.read(file.getData())).keys[0];
+          prv = (await KeyUtil.parseBinary(file.getData()))[0];
```

Comment:
I may input a file that contains OpenPGP armored key. Please add this test case - since it didn't stop working, we're probably not testing it.

Reply:
The `if` from that `else` takes care of armored input but this has issues for X.509 private keys (the code takes first armored block and in X.509 case we need all of them). No worries I'll see what we can do in this case and add test cases along the way.

Reply:
I see it now, thank you.